activestorage/lib/active_storage/attached/model.rb

```diff
@@ -75,7 +75,9 @@ module Attached::Model
       # proxy that provides the dynamic proxy to the associations and factory methods, like +attach+.
       #
       # If the +:dependent+ option isn't set, the attachment will be purged
-      # (i.e. destroyed) whenever the record is destroyed.
+      # (i.e. destroyed) whenever the record is destroyed. The +:dependent+ option  by default is +:purge_later+.
+      # This means that a job will be enqueued to destroy the blob utilizing a ActiveJob::Backend queue adapter.
+      # If a queue adapter is not set in the application, it will raise an error.
```

Comment:
I think we can remove some duplication in explaining the default option.
We can also recommend an alternative for when the queue adapter isn't set.
Maybe something like:
```suggestion
      # The +:dependent+ option defaults to +:purge_later+. This means the attachment will be
      # purged (i.e. destroyed) in the background whenever the record is destroyed. 
      # If an ActiveJob::Backend queue adapter is not set in the application set it to
      # +purge+ instead.
```

This also applies to `has_many_attached`.